pkgs/applications/misc/qt-box-editor/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, fetchFromGitHub
+, qtbase
+, qtsvg
+, qmake
+, leptonica
+, tesseract
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qt-box-editor";
+  version = "unstable-2019-07-12";
+
+  src = fetchFromGitHub {
+    owner = "zdenop";
+    repo = "qt-box-editor";
+    rev = "75a68b466868ba41ba2886caa796057403fe1901";
+    sha256 = "0zwsyy7cnbhy5aazwlkhd9y8bnzlgy1gffqa46abajn4809b95k3";
+  };
+
+  buildInputs = [ qtbase qtsvg leptonica tesseract ];
+
+  nativeBuildInputs = [ qmake ];
+
+  preConfigure = ''
+    printf "INSTALLS += target\ntarget.path = $out/bin" >>  qt-box-editor.pro
```

Comment:
Can you submit this as a patch upstream?

In the patch `target.path` should be
```
target.path = $$PREFIX/bin
```

Reply:
Here is a PR in qt-box-editor to fix the issue. I have tested that on the next release it will working without the postConfigure step. https://github.com/zdenop/qt-box-editor/pull/78